docs/content/how-to-quickly-install-the-temporal-server.md

```diff
@@ -0,0 +1,36 @@
+---
+id: how-to-quickly-install-the-temporal-server
+title: How to quickly install the Temporal Server for testing and local development
+description: Use `docker-compose` to quickly install and run the Temporal Server locally to test the system while developing Workflows.
+tags:
+  - guide
+---
+
+import RelatedReadList from '../components/RelatedReadList.js'
+
+Use `docker-compose` to quickly install and run the Temporal Server locally to test the system while developing Workflows.
+
+Make sure both Docker and `docker-compose` are installed.
+
+- [Install Docker](https://docs.docker.com/engine/install)
+- [Install docker-compose](https://docs.docker.com/compose/install)
+
+Then clone the [temporalio/docker-compose](https://github.com/temporalio/docker-compose) repository and run `docker-compose up` from the root of that repo:
+
+```bash
+git clone https://github.com/temporalio/docker-compose.git
+cd  docker-compose
+docker-compose up
+```
+
+Once the Temporal Server is running, Workflow Executions can be invoked.
```

Comment:
```suggestion
When the Temporal Server is running, Workflow Executions can be invoked.
```

Per Google style guide for "once" (although "When" seems a better fit than "After" here).